tutorials/singularity-containers-with-cloud-build/README.md

```diff
@@ -7,7 +7,9 @@ date_published: 2019-01-18
 ---
 
 This tutorial shows you how to use [Cloud Build](https://cloud.google.com/cloud-build/) to build [Singularity](https://www.sylabs.io/singularity/) containers. 
-In constrast to [Docker](https://www.docker.com/), the Singularity container binary is designed specifically for HPC workloads. 
+In constrast to [Docker](https://www.docker.com/), the Singularity container binary is designed specifically for HPC workloads.
+
+This diagram illustrates the process ![diagram](overview.png)
```

Comment:
Wow these diagrams are beautiful!

For the cloudbuild.yaml, are you able to tap him over a little to the left? So the text isn't broken?

![image](https://user-images.githubusercontent.com/814322/52666445-2d003700-2ec3-11e9-9493-8396192e7e99.png)

Reply:
Nice catch. Done.

Reply:
Looks good!